lib/SizeFormatHelpers.js

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
You could go either way on this. I just created this to have a place for other size based formatters to live specific to webpack.